modules/web/js/ballerina/components/fork-join-statement.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from "react";
+import BlockStatementDecorator from "./block-statement-decorator";
+import CompoundStatementDecorator from "./compound-statement-decorator";
+import PropTypes from 'prop-types';
+import {statement, blockStatement} from './../configs/designer-defaults';
+import {getComponentForNodeArray} from './utils';
+import SimpleBBox from './../ast/simple-bounding-box';
+
+class ForkJoinStatement extends React.Component {
+
+    render() {
+        let model = this.props.model,
+            bBox = model.viewState.bBox,
+            bodyBBox = model.viewState.components.body;
+        const children = getComponentForNodeArray(this.props.model.getChildren());
+
+
+        const forkBBox = new SimpleBBox(bBox.x, bBox.y + statement.gutter.v, bBox.w, bodyBBox.h
+            + blockStatement.heading.height);
+        return (<CompoundStatementDecorator model={model} bBox={bBox}>
+            <BlockStatementDecorator hideLifeLine={true} dropTarget={model} bBox={forkBBox} title={"Fork"}>
```

Comment:
We are avoiding the use of double quotes as per our lint rules. You can fix them using , 

`npm run lintfile -- <filename> --fix`